content/manager/upload-blueprint.md

```diff
@@ -6,43 +6,42 @@ draft: false
 weight: 400
 ---
 
-Before Cloudify Manager can deploy your blueprint, you must upload it. You can upload a blueprint using the CLI or (for Premium users) the Cloudify Web interface.
+Before you can deploy a blueprint, you must upload the blueprint to the Cloudify Manager. You can upload a blueprint using the CLI. Premium users can also upload using the Cloudiy Web Interface.
 
 Either use a blueprint that you have written or download an [example blueprint](https://github.com/cloudify-cosmo/cloudify-nodecellar-example) to upload.
 
-## Uploading via the CLI
 
-From the Cloudify command-line interface, you can upload your blueprint to Cloudify Manager. You must specify a path to a blueprint file. Cloudify compresses the folder and its contents.
+## Uploading a Blueprint using the Cloudify Web Interface
 
-The following is an example of `upload`:
-{{< gsHighlight  bash >}}
-$ cfy blueprints upload -b BLUEPRINT_ID -p BLUEPRINT_FILE_LOCATION
-...
-
-...
-{{< /gsHighlight >}}
-
-
-## Uploading a Blueprint via the Cloudify Web Interface
-
-If you are a Premium version user, you can upload a pre-packaged blueprint archive, such as *.tar, *.tar.gz, *.tar.bz, *.zip., using the Cloudify Manager UI.
+If you are a Premium version user, you can upload a pre-packaged blueprint archive using the Cloudify Web Interfacein tar, tar.gz, tar.bz, or zip formats.
```

Comment:
Again, not just for premium users (starting 4.2).

Reply:
This is for 4.1.

Reply:
Right again